core/shared/src/main/scala/doodle/effect/Writer.scala

```diff
@@ -17,17 +17,6 @@
 package doodle
 package effect
 
-import cats.effect.IO
 import doodle.algebra.Algebra
-import doodle.algebra.Picture
-import doodle.core.format.Format
 
```

Comment:
```suggestion
/** Marker trait for effects that write a picture to a file or some other type. */
```

Adding basic ScalaDoc, especially for the parts of the code that are user level concepts, is a good practice.